reflection.md

```diff
@@ -17,3 +19,7 @@ Write a few brief paragraphs that discuss:
   implementing it? (You don't have to write code for this part. You could say
   "I would use extension to..." or "Scala's rules for valid
   identifiers prevent...")
+
+I thought most of the new operators were much easier to use in this design. It feels much more natural to be able to use the operators than to write a bunch of methods. That being said, sometimes sequences of operators are harder to read than names of methods that are very clear what they do.
```

Comment:
I agree with this point that you brought up. It makes things seem as if there's *some* tradeoff between concision and readability, although might just be because neither of us are exposed often to the symbols used for regular expressions.